src/StorageMover/StorageMover.Autorest/test/New-AzStorageMoverProject.Recording.json

```
@@ -17,39 +17,42 @@
-        "x-ms-ratelimit-remaining-subscription-writes": [ "1187" ],
+        "ETag": [ "\"0000f136-0000-3400-0000-66692a000000\"" ],
+        "x-ms-throttling-version": [ "v1" ],
         "x-ms-providerhub-traffic": [ "True" ],
-        "mise-correlation-id": [ "35830e65-e633-4911-a33f-dc7af35e734b" ],
-        "x-ms-request-id": [ "39a7ceb2-78ac-4c01-b03d-176adf28c0e5" ],
-        "x-ms-correlation-request-id": [ "a8fd9d27-9e5d-41b4-8cb3-7a48a48c1c02" ],
-        "x-ms-routing-request-id": [ "EASTUS2EUAP:20230721T042929Z:a8fd9d27-9e5d-41b4-8cb3-7a48a48c1c02" ],
+        "mise-correlation-id": [ "d5b12637-c892-4476-9468-e5758d37d070" ],
+        "x-ms-request-id": [ "220a0e95-5b92-4dae-a4aa-f57756d63e4c" ],
+        "x-ms-correlation-request-id": [ "fe48e2cc-675e-484a-8102-f952d0951157" ],
+        "x-ms-ratelimit-remaining-subscription-writes": [ "1199" ],
+        "x-ms-routing-request-id": [ "SOUTHEASTASIA:20240612T045425Z:fe48e2cc-675e-484a-8102-f952d0951157" ],
         "Strict-Transport-Security": [ "max-age=31536000; includeSubDomains" ],
         "X-Content-Type-Options": [ "nosniff" ],
-        "Date": [ "Fri, 21 Jul 2023 04:29:28 GMT" ]
+        "X-Cache": [ "CONFIG_NOCACHE" ],
+        "X-MSEdge-Ref": [ "Ref A: 6668A3BFA5D64EBDB58D474C43948F1E Ref B: MAA201060516019 Ref C: 2024-06-12T04:54:23Z" ],
+        "Date": [ "Wed, 12 Jun 2024 04:54:25 GMT" ]
       },
       "ContentHeaders": {
-        "Content-Length": [ "640" ],
+        "Content-Length": [ "602" ],
         "Content-Type": [ "application/json; charset=utf-8" ],
         "Expires": [ "-1" ]
       },
-      "Content": "{\"id\":\"/subscriptions/45b60d85-fd72-427a-a708-f994d26e593e/resourceGroups/teststoragemover/providers/Microsoft.StorageMover/storageMovers/testStoMover1hc715gxy/projects/testProjecthc715gxy\",\"name\":\"testProjecthc715gxy\",\"type\":\"microsoft.storagemover/storagemovers/projects\",\"systemData\":{\"createdBy\":\"a000255f-5f09-45e0-a970-9d9ed9cc6453\",\"createdByType\":\"Application\",\"createdAt\":\"2023-07-21T04:28:58.1006856Z\",\"lastModifiedBy\":\"a000255f-5f09-45e0-a970-9d9ed9cc6453\",\"lastModifiedByType\":\"Application\",\"lastModifiedAt\":\"2023-07-21T04:29:29.4609543Z\"},\"properties\":{\"provisioningState\":\"Succeeded\",\"description\":\"test project description\"}}",
+      "Content": "{\"id\":\"/subscriptions/45b60d85-fd72-427a-a708-f994d26e593e/resourceGroups/teststoragemover/providers/Microsoft.StorageMover/storageMovers/testStoMover1pul1qe2g/projects/testProjectpul1qe2g\",\"name\":\"testProjectpul1qe2g\",\"type\":\"microsoft.storagemover/storagemovers/projects\",\"systemData\":{\"createdBy\":\"yifanzhang@microsoft.com\",\"createdByType\":\"User\",\"createdAt\":\"2024-06-12T04:53:12.2543326Z\",\"lastModifiedBy\":\"yifanzhang@microsoft.com\",\"lastModifiedByType\":\"User\",\"lastModifiedAt\":\"2024-06-12T04:54:23.8146635Z\"},\"properties\":{\"provisioningState\":\"Succeeded\",\"description\":\"test project description\"}}",
```

```suggestion
      "Content": "{\"id\":\"/subscriptions/45b60d85-fd72-427a-a708-f994d26e593e/resourceGroups/teststoragemover/providers/Microsoft.StorageMover/storageMovers/testStoMover1pul1qe2g/projects/testProjectpul1qe2g\",\"name\":\"testProjectpul1qe2g\",\"type\":\"microsoft.storagemover/storagemovers/projects\",\"systemData\":{\"createdBy\":\"yifanzhang@microsoft.com\",\"createdByType\":\"User\",\"createdAt\":\"2024-06-12T04:53:12.2543326Z\",\"lastModifiedBy\":\"placeholder@mail.com\",\"lastModifiedByType\":\"User\",\"lastModifiedAt\":\"2024-06-12T04:54:23.8146635Z\"},\"properties\":{\"provisioningState\":\"Succeeded\",\"description\":\"test project description\"}}",
```

By replacing a email with palceholder, we can avoid PIIs included in a public repo.

---

Done. Replaced email address with placeholder in all recording files.